diesel_codegen/src/queryable.rs

```diff
@@ -41,7 +41,7 @@ pub fn expand_derive_queryable(
 
         let impl_item = quote_item!(cx,
             impl$generics ::diesel::Queryable<__ST, __DB> for $ty where
```

Comment:
This was mentioned privately, but just so it's documented publicly:

The double underscores are because we inhabit the same namespace as user code, and I always put some underscores when we are able to cause namespace conflicts. I also prefer to put `diesel` in the name somewhere if it's ever user visible (in this case it isn't). Sam pointed out that `DIESEL_DB` would be just as good but we basically dropped it there because it's a non-issue.